git/operations.go

```diff
@@ -137,12 +137,26 @@ func revlist(path, ref string) ([]string, error) {
 }
 
 // Return the revisions and one-line log commit messages
-func onelinelog(path, refspec string) ([]Commit, error) {
+func onelinelog(path, refspec, gitpath string) ([]Commit, error) {
```

Comment:
I would interpret `gitpath` as meaning "path to git", and hence or otherwise not associate it with a subdirectory of the repo. Perhaps something like `subdir` would be better?

Reply:
We have been inconsistent elsewhere, which doesn't help :/

Reply:
Yes, I chose "gitpath" for the association with the gitpath flag. Will rename it to subdir and put a comment.